src/ORM/Behavior/TimestampBehavior.php

```diff
@@ -136,8 +136,9 @@ public function implementedEvents()
      */
     public function timestamp(DateTime $ts = null, $refreshTimestamp = false)
     {
-        if ($ts) {
-            if ($this->_config['refreshTimestamp']) {
+        if ($ts instanceof DateTime) {
```

Comment:
I'd say if changing this all, then it should be an explicit not-`null` comparison, as the argument typhint restricts things to either `DateTime` or `null`, ie no other value can get there in the first place.